src/systems/tips.js

```diff
@@ -82,263 +37,62 @@ const tipPlatform = () => {
-  window.APP.store.resetConfirmedBroadcastedRooms();
-  finishedScopes = {};
-};
+}
 
 const VALIDATORS = {
   look: function(userinput) {
-    if (
-      userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-      userinput.activeSets.includes(sets.leftCursorHoldingPen)
-    )
-      return INVALID;
     const cameraDelta = userinput.get(
       isMobile ? paths.device.touchscreen.touchCameraDelta : paths.device.smartMouse.cameraDelta
     );
     return cameraDelta ? FINISH : VALID;
   },
   locomotion: function(userinput) {
-    if (
-      userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-      userinput.activeSets.includes(sets.leftCursorHoldingPen)
-    )
-      return INVALID;
     const accel = userinput.get(paths.actions.characterAcceleration);
 
     // User moved
     return accel && (accel[0] !== 0 || accel[1] !== 0) ? FINISH : VALID;
   },
   turning: function(userinput) {
-    if (
-      userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-      userinput.activeSets.includes(sets.leftCursorHoldingPen)
-    )
-      return INVALID;
     if (userinput.get(paths.actions.snapRotateLeft) || userinput.get(paths.actions.snapRotateRight)) return FINISH;
     return VALID;
   },
-  spawn_menu: function(userinput, scene, mediaCounter) {
-    if (
-      userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-      userinput.activeSets.includes(sets.leftCursorHoldingPen)
-    )
-      return INVALID;
-    if (mediaCounter.count() === 0) return VALID;
-    return FINISH;
-  },
-  freeze_gesture: function(userinput, scene, mediaCounter) {
-    if (mediaCounter.count() === 0) return INVALID;
-    if (userinput.activeSets.includes(sets.rightCursorHoldingInteractable)) return INVALID;
-    if (userinput.activeSets.includes(sets.rightCursorHoldingPen)) return INVALID;
-    if (userinput.activeSets.includes(sets.leftCursorHoldingInteractable)) return INVALID;
-    if (userinput.activeSets.includes(sets.leftCursorHoldingPen)) return INVALID;
-    if (
-      scene.is("frozen") &&
-      (userinput.activeSets.includes(sets.rightCursorHoveringOnInteractable) ||
-        userinput.activeSets.includes(sets.leftCursorHoveringOnInteractable))
-    )
-      return FINISH;
-    return scene.is("frozen") ? INVALID : VALID;
-  },
-  menu_hover: function(userinput, scene, mediaCounter) {
-    if (mediaCounter.count() === 0) return INVALID;
-    if (!scene.is("frozen")) return INVALID;
-    if (userinput.activeSets.includes(sets.rightCursorHoldingPen)) return INVALID;
-    if (userinput.activeSets.includes(sets.leftCursorHoldingPen)) return INVALID;
-    if (
-      scene.is("frozen") &&
-      (userinput.activeSets.includes(sets.rightCursorHoveringOnInteractable) ||
-        userinput.activeSets.includes(sets.leftCursorHoveringOnInteractable))
-    )
-      return FINISH;
-    return VALID;
-  },
-  invite: function(userinput, scene, _mediaCounter, _store, hub) {
+  invite: function(_userinput, scene, hub) {
```

Comment:
I think the FINISH step for this should gated on pressing the invite button - not on becoming copresent.

Reply:
I'm probably not going to change this just now since it's new functionality. But I tend to agree.